packages/ra-ui-materialui/src/layout/Sidebar.tsx

```diff
@@ -45,22 +55,21 @@ const useStyles = makeStyles(
     { name: 'RaSidebar' }
 );
 
-const Sidebar = props => {
-    const {
-        children,
-        closedSize,
-        size,
-        classes: classesOverride,
-        ...rest
-    } = props;
+interface Props extends DrawerProps {
+    children: ReactElement;
```

Comment:
I think it should be `React.ReactNode`

Reply:
ReactNode gives an error

`Type 'string' is not assignable to type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)>) | (new (props: any) => Component<any, any, any>)>'.`